includes/ziti/ziti_model.h

```diff
@@ -20,6 +20,15 @@ limitations under the License.
 
 #include "model_support.h"
 
+#define ZITI_AUTH_QUERY_MFA_MODEL(XX, ...) \
+XX(type_id, string, none, typeId, __VA_ARGS__) \
```

Comment:
is this an enum? also `provider`?

Reply:
It will be, right now we have only 1 value.